app/views/movies/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @movie.title %></h1>
+<p>Publish Year: <%= @movie.creation_year %></p>
+<p>Genre: <%= @movie.genre %></p>
+<h3>Actors:</h3>
+<% @movie.actors_by_age.each do |actor| %>
+  <li><%= actor %></li>
+<% end %>
+
+<h3>Average Actor Age</h3>
+<%= @movie.average_actor_age %>
+
+<h3><u>Add an Actor!</u></h3>
+<%= form_with method: :get, local: :true do |f| %>
```

Comment:
I'm not sure I agree with the `method: :get` here, since the user story is really asking us to **create a relationship** between this movie and an actor. The associated HTTP method to a create is `post`. Using a `GET` method would work if we're only searching for an actor, but since we're creating a relationship, we should use `:post`.